### ISSUE TITLE
doc: Syntax doc not accessible to screen readers

### ISSUE THREAD

**Issue by msudberg** [MaterializeInc/materialize#6737]
We document supported SQL syntax with SVG images, which aren't accessible to people using screen readers and may prove difficult for some other visual impairments. We need a textual alternative for ADA compliance.
